index.html

```diff
@@ -106,6 +113,9 @@ <h1 class="header">♣️ Basics - Blackjack ♠️</h1>
 
         // Reset input value
         input.value = "";
+
+        // Edit the title text to a new value
+        document.querySelector("#input-title").innerHTML = inputDisplay();
```

Comment:
this is interesting. Is there any reason why you query input title with getElementById on line 1-2 and querySelector on line 118